docs/community/developer_call_notes/index.rst

```diff
@@ -0,0 +1,11 @@
+:orphan:
+
+.. _developer_call_notes:
+
+Developer call notes
+--------------------
+
+.. toctree::
+   :maxdepth: 1
+
+   2020_may_7
```

Comment:
This does not sort well. Let's do 2020_05_07 [by beginning with a year, there should be no confusion about month/day]

Reply:
This is just as a file name convention, the header in the file can say May 7, 2020 as you do now.